src/modules/services/components/FilePicker/sharing.js

```
@@ -20,65 +20,58 @@ const getShortcode = permission => {
-  try {
-    const permissionsCol = client.collection('io.cozy.permissions')
-    const result = await permissionsCol.findLinksByDoctype('io.cozy.files')
-    const permissions = result?.data ?? []
-
-    const existingPermission = permissions.find(perm =>
-      isPermissionRelatedTo(perm, file)
+  const existingPermission = permissions.find(permission => {
+    const fileIds = permission.attributes?.permissions?.files?.values ?? []
+    return (
+      fileIds.includes(fileId) && (!singleFileOnly || fileIds.length === 1)
     )
```

ESLint

[error] Replace `(⏎······fileIds.includes(fileId)·&&·(!singleFileOnly·||·fileIds.length·===·1)⏎····` with `fileIds.includes(fileId)·&&·(!singleFileOnly·||·fileIds.length·===·1`

(prettier/prettier)